plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -376,6 +383,35 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 					}
 				}
 
+	if (devices & Debuginator_Mouse) {
+		CApiInputControllerPtr mouse = script_api->Input->mouse();
+		input_wrapper.controller = mouse;
+
+		if (api->any_released(input_wrapper.controller) != UINT_MAX) {
+			input_wrapper.time_since_pressed = -1;
+		}
+
+
+		if (debuginator_is_open(debuginator)) {
+			CApiVector3 scroll = input_wrapper.axis("wheel");
+			if (scroll.y != 0) {
+				debuginator->focus_height += scroll.y * 0.05f;
```

Comment:
I'm not sure about this solution.. The idea with focus_height is to say "I want the hot item to be at 30% from the top of the screen". Modifying it based on scrolling, presumably getting it beyond [0,1] range feels a bit strange..

Wouldn't it be more reasonable to change the hot item? Then everything would be taken care of "automatically".